packages/core/src/extensions/common-api/utils.ts

```diff
@@ -7,20 +7,23 @@ import openLinkInBrowserInjectable from "../../common/utils/open-link-in-browser
 import buildVersionInjectable from "../../main/vars/build-version/build-version.injectable";
 import { asLegacyGlobalFunctionForExtensionApi } from "../as-legacy-globals-for-extension-api/as-legacy-global-function-for-extension-api";
 import { getLegacyGlobalDiForExtensionApi } from "../as-legacy-globals-for-extension-api/legacy-global-di-for-extension-api";
-import { Singleton } from "../../common/utils";
-import { prevDefault, stopPropagation } from "../../renderer/utils/prevDefault";
-import { cssNames } from "../../renderer/utils/cssNames";
-
-export const Util = {
-  Singleton,
-  prevDefault,
-  stopPropagation,
-  cssNames,
-  openExternal: asLegacyGlobalFunctionForExtensionApi(openLinkInBrowserInjectable),
-  openBrowser: asLegacyGlobalFunctionForExtensionApi(openLinkInBrowserInjectable),
-  getAppVersion: () => {
-    const di = getLegacyGlobalDiForExtensionApi();
-
-    return di.inject(buildVersionInjectable).get();
-  },
-} as const;
+
+export { Singleton } from "../../common/utils";
+export { prevDefault, stopPropagation } from "../../renderer/utils/prevDefault";
```

Comment:
no longer exported as part of `Util`?

Reply:
No they are, I changes the use from `{ Util }` to `* as Util`

Reply:
ah right, in line with other types like this.